downstream/modules/platform/proc-add-operator-execution-nodes.adoc

```diff
@@ -10,9 +10,9 @@
 . In the navigation panel, select menu:Administration[Instances].
 . Click btn:[Add].
 . Input the VM name in the *Host Name* field.
-. Optional: Input the port number in the *Listner Port* field.
+. Optional: Input the port number in the *Listener Port* field.
 . Click btn:[Save].
-. Click the download icon image:download.png[download]next to *Install Bundle*. This will start a download, take note of where you save the file
+. Click the download icon image:download.png[download,15,15]next to *Install Bundle*. This will start a download, take note of where you save the file
```

Comment:
```suggestion
. Click the download icon image:download.png[download,15,15]next to *Install Bundle*. This starts a download, take note of where you save the file.
```

Reply:
Just a small tip as per the SG, avoid using future tense when possible:
https://docs.engineering.redhat.com/display/CCSENABLE/Future+tense